src/txorphanage.cpp

```diff
@@ -278,7 +278,7 @@ bool TxOrphanage::HaveTxToReconsider(NodeId peer)
     return false;
 }
 
-void TxOrphanage::EraseForBlock(const CBlock& block)
+std::vector<uint256> TxOrphanage::EraseForBlock(const CBlock& block)
```

Comment:
unused in this PR, but is exercised in tests so I'm happy for it to be used later :+1: